vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -407,10 +466,10 @@ private void loadExperimentInfo(Experiment experiment) {
     this.experimentSummary.removeAll();
     this.experimentSummary.add(factSheet);
     factSheet.showMenu();
-    reloadExperimentalVariables(experiment);
+    reloadExperimentInformation(experiment);
   }
 
-  private void reloadExperimentalVariables(Experiment experiment) {
+  private void reloadExperimentInformation(Experiment experiment) {
```

Comment:
why rename the method? as far as I can see this still only reloads the variables

Reply:
Oh good point I changed it back 👍